manage.py

```diff
@@ -14,5 +15,15 @@ def make_shell_context():
 manager.add_command("shell", Shell(make_context=make_shell_context))
 manager.add_command('db', MigrateCommand)
 
+#define a command for running tests
```

Comment:
good implementation. Does it work when you run it?

Reply:
yes it works